Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "oasis-cbor"
 description = "Rust CBOR serialization built as a thin wrapper around sk-cbor."
-version = "0.1.8"
+version = "0.1.9"
```

Comment:
Should the version be bumped to `0.2.0` since afaik https://github.com/oasisprotocol/oasis-cbor/pull/20 was a breaking change? (i had to update oasis-cbor in all dependencies (e.g. oasis-core) when testing this in oasis-sdk)).

Reply:
Ok, bumped to 0.2.0.